src/frontend/src/lib/components/hero/Header.svelte

```diff
@@ -11,13 +11,10 @@
 	import { modalAboutHow, modalAboutWhat } from '$lib/derived/modal.derived';
 
 	export let back = false;
-
-	let mw: 'sm' | 'xl';
-	$: mw = $authSignedIn ? 'sm' : 'xl';
 </script>
 
 <header
-	class={`grid grid-cols-2 ${mw}:grid-cols-[1fr_auto_1fr] items-center md:px-4 relative z-10 pointer-events-none`}
+	class={`grid grid-cols-2 ${$authSignedIn ? 'sm:grid-cols-[1fr_auto_1fr]' : 'xl:grid-cols-[1fr_auto_1fr]'} items-center md:px-4 relative z-10 pointer-events-none`}
```

Comment:
I would use conditional class then.

```
<header
   class="grid grid-cols-2..."
   class:sm:grid-cols-[1fr_auto_1fr]={$authSignedIn}
   class:xl:grid-cols-[1fr_auto_1fr]={$authNotSignedIn}
```

WDYT?

Reply:
ah yes! maybe a bit verbose but more organized

Reply:
yeah. ultimately tailwind is bit verbose anyway...

Reply:
done